app/assets/javascripts/fae/navigation/_subnav_highlighter.js

```diff
@@ -69,21 +69,10 @@ Fae.navigation.subnav_highlighter = {
 
       $('.js-highligher').last().addClass('-active').removeClass('js-highligher');
     }
-    FCH.scroll.push(scrollCallback);
 
-    //highlight the first one on page load
+    // highlight the first one on page load
     scrollCallback();
-
-    /**
-     * On resize, ensure last section is the same as the window height so when we skip to it the label is at the top
-     * @private
-     */
-    function lastSectionBuffer() {
```

Comment:
We've had internal discussion about this and determined the space was more of an eye sore than not scrolling to the top of the section when using the nav. You do make a good point about underlining the section, which we could potentially fix by underlining the last item whenever you're at the bottom of the page.